Makefile

```diff
@@ -125,3 +128,15 @@ android-ports: ##@other Add reverse proxy to Android Device/Simulator
 	adb reverse tcp:8081 tcp:8081
 	adb reverse tcp:3449 tcp:3449
 	adb reverse tcp:4567 tcp:4567
+
+start-dev:
+	${MAKE} prepare
+	${MAKE} prod-build
```

Comment:
Why do you need that for dev?

Reply:
I wasn't sure, I thought it was not needed, but I was just following the wiki https://wiki.status.im/Building_Status, happy to remove it from both if not needed.

Reply:
For dev you only need the npm/re-natal dance then figwheel running.